### ISSUE TITLE
proton-call: error: 'PC_COMMON' does not exist

### ISSUE THREAD

**Issue by Sojiro84** [caverym/Proton-Caller#15]
I have set up the **proton.conf** at **.config** with the following content:

```
data = "/home/sojiro/Documents/proton-caller/data/"
common = "/home/sojiro/.steam/steam/steamapps/common/"
```

When I want to launch a program with **proton-call 5.13 Wow.exe** I get the following error:

_proton-call: error: 'PC_COMMON' does not exist_

The folder path's are valid. What does the error mean?

**Comment by caverym** [comment]
Make sure your the program is updated to 2.2.x, also make sure all the directories exist.

**Comment by Sojiro84** [comment]
That was it indeed! Problem fixed with 2.2!
